scripts/pull-from-quay

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+if [[ -z "$2" ]]; then
+  echo "Usage: pull-from-quay [NAMESPACE] [REPOSITORY]"
```

Comment:
Nit. use `$0` to refer to the script itself to get accurate path to it: `"Usage: $0 [NAMESPACE] [REPOSITORY]"`

Reply:
That's great feedback! Thanks!

Reply:
Resolved in the latest commit.